docs/reference/filebeat/delete-file-guide.md

```diff
@@ -0,0 +1,171 @@
+# Removing files after ingestion
+
+::::{warning}
+Enabling this feature will remove files, which could lead to unintentional data loss if not configured correctly.
+::::
+
+The Filestream input can remove files after they have been fully
+ingested. Three requirements need to be met before the Filestream
+input can remove a file:
```

Comment:
```suggestion
ingested. Filestream input can remove a file only if one of the following conditions apply:
```

Reply:
That's actually changing the meaning @theletterf. All three requirements need to be met for the file to be removed. Maybe:
```
Filestream input can remove a file only if all of the following conditions apply:
```